src/main/java/com/foundation/trello/util/ReadConfiguration.java

```diff
@@ -41,6 +43,9 @@ private Properties readConfigurationFile() {
     }
 
     /**
+     * This method ensures that only one instance is created according
+     * to the builder pattern.
+     *
      * @return an instance.
```

Comment:
Please explain the content and use of returned instance

Reply:
Solved.